include/gridtools/sid/composite.hpp

```diff
@@ -137,7 +137,7 @@ namespace gridtools {
 
                 struct sum {
                     template <class Lhs, class Rhs>
-                    GT_FUNCTION constexpr auto operator()(Lhs &&lhs, Rhs &&rhs) const {
+                    GT_FORCE_INLINE constexpr auto operator()(Lhs &&lhs, Rhs &&rhs) const {
```

Comment:
I remove host/device: it's used in both host and device context, but potentially calls a function which is only host (which is ok, as it only happens in a context where the callee of this function is host only). Now we rely on `constexpr` to work correctly in this case.

Reply:
But then we can drop `sum` because it’s identical to `std::plus<void>`, isn’t it?

Reply:
seems to work